framework/wazuh/core/cluster/server.py

```diff
@@ -230,6 +283,102 @@ def __init__(self, performance_test: int, concurrency_test: int, configuration:
         self.tasks = [self.check_clients_keepalive]
         self.handler_class = AbstractServerHandler
         self.loop = asyncio.get_running_loop()
+        self.broadcast_results = {}
+
+    def broadcast(self, f, *args, **kwargs):
+        """Add a function to the broadcast_queue of each server handler.
+
+        Parameters
+        ----------
+        f : Callable
+            Function to be run in each server handler.
+        *args
+            Arguments to be passed to function `f`.
+        **kwargs
+            Keyword arguments to be passed to function `f`.
+
+        Notes
+        -----
+        This method does not allow determining whether the function has been
+        executed in all server handlers or the result for each one. For those
+        features, see `broadcast_add` and `broadcast_pop`.
+        """
+        for name, client in self.clients.items():
+            try:
+                client.add_request(None, f, *args, **kwargs)
+                self.logger.debug2(f'Added broadcast request to execute "{f.__name__}" in {name}.')
+            except Exception as e:
+                self.logger.error(f'Error while adding broadcast request in {name}: {e}', exc_info=False)
+
+    def broadcast_add(self, f, *args, **kwargs):
+        """Add a function to the broadcast_queue of each server handler and obtain an identifier.
+
+        Parameters
+        ----------
+        f : Callable
+            Function to be run in each server handler.
+        *args
+            Arguments to be passed to function `f`.
+        **kwargs
+            Keyword arguments to be passed to function `f`.
+
+        Returns
+        -------
+        broadcast_id : str
+            Identifier to check the status of the broadcast request.
+
+        Notes
+        -----
+        It is important to run `broadcast_pop` to remove the result entry from the
+        broadcast_results dict after using this method. Otherwise, it will be kept
+        until restarting the server. See `broadcast` method if broadcast results
+        are not needed.
+        """
+        if self.clients:
+            broadcast_id = str(uuid4())
+            self.broadcast_results[broadcast_id] = {}
+
+            for name, client in self.clients.items():
+                try:
+                    self.broadcast_results[broadcast_id][name] = NO_RESULT
```

Comment:
```suggestion
                    self.broadcast_results[broadcast_id][name] = AbstractServer.NO_RESULT
```

Add No_RESULT to AbstractServer as class variable